### PR TITLE
More Adopters

### LINKED ISSUE #3: Add full listing of adopters
There is a list of adopters in [this Markdown file](https://github.com/tikv/tikv/blob/master/docs/adopters.md). That list needs to be vetted and transferred into the `data/adopters.yaml` file in this repo.

### PR DESCRIPTION
**Pull request by Hoverbear**
Closes #3 .

### CODE REVIEW

**Comment by lilin90** [comment]
@CaitinChen Please help check the newly added adopters. Thanks!

File: data/adopters.yaml [hunk 1]
```diff
@@ -36,3 +36,274 @@
...
+- name: Haian eKing Technology
+  url: https://www.crunchbase.com/organization/hainan-eking-technology
+  industry: Enterprise Technology
+  stories:
+  - language: Chinese
+    url: https://pingcap.com/cases-cn/user-case-ekingtech/
+- name: LianLian Tech
+  url: http://www.10030.com.cn/web/
+  industry: Mobile Payment
+- name: TongDun Technology
```
Comment (CaitinChen) [inline data/adopters.yaml]:
```suggestion
- name: Tongdun Technology
```

File: data/adopters.yaml [hunk 2]
```diff
@@ -36,3 +36,274 @@
...
+- name: Mashang Consumer Finance
+  url: https://www.crunchbase.com/organization/ms-finance
+  industry: FinTech
+- name: Tencent OMG
+  url: https://en.wikipedia.org/wiki/Tencent
+  industry: Media
+- name: Terren
+  url: http://webterren.com.zigstat.com/
+  industry: Media
+- name: Lecloud
```
Comment (CaitinChen) [inline data/adopters.yaml]:
```suggestion
- name: LeCloud
```

File: data/adopters.yaml [hunk 3]
```diff
@@ -36,3 +36,274 @@
...
+  industry: Finance
+- name: Wuhan Antian Information Technology
+  url: https://www.avlsec.com/
+  industry: Enterprise Technology
+- name: Ausnutria Dairy
+  url: https://www.crunchbase.com/organization/ausnutria-dairy
+  industry: FoodTech
+- name: Qingdao Telaidian
+  url: https://www.teld.cn/
+  industry: Electric Car Chargers
```
Comment (CaitinChen) [inline data/adopters.yaml]:
```suggestion
  industry: Electric Car Charger
```

File: data/adopters.yaml [hunk 4]
```diff
@@ -36,3 +36,274 @@
     url: https://www.pingcap.com/blog/use-case-tidb-in-eleme/
   - language: Chinese
     url: https://www.pingcap.com/cases-cn/user-case-eleme-1/
+- name: Xiaomi
+  url: https://www.mi.com/
```
Comment (CaitinChen) [inline data/adopters.yaml]:
```suggestion
  url: https://en.wikipedia.org/wiki/Xiaomi
```

Comment (CaitinChen) [reply data/adopters.yaml]: Sorry, the link in adopters.md of the docs repo is wrong. I will also change the link in the docs repo.
We prefer the Wikipedia link here. If the company Wikipedia link doe not exist, we will use its Crunchbase link. If the the company Crunchbase link does not exist, we will use the official website link of this company.

**Comment by Hoverbear** [comment]
@CaitinChen PTAL again. :)

Thanks @dcalvin

**Pull request merged**